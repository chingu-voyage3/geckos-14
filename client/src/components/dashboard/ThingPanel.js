/** ThingPanel Component :
The ThingPanel displays  a controlPanel and a ThingList
ControPanel needs the params, actions and parent props
ThingList just needs the list of devices
 */

import React from 'react';
import ThingList from './thing/ThingList';
import ControlPanel from './panel/ControlPanel';

const ThingPanel = props => {
  return (
    <div className="thing-panel">
      <ControlPanel {...props} />
      <ThingList {...props} />
    </div>
  );
};

export default ThingPanel;
