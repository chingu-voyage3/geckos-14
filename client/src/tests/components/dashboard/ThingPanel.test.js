import React from 'react';
import ThingPanel from '../../../components/dashboard/ThingPanel';

import { shallow } from 'enzyme';

it('renders without crashing', () => {
  const wrapper = shallow(<ThingPanel />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
