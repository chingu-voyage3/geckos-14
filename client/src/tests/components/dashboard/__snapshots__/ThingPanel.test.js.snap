// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="thing-panel"
>
  <ControlPanel />
  <ThingList />
</div>
`;
